### DISCUSSION TITLE
GitHub Pages not working for my specific repository for some reason

### DISCUSSION THREAD

**Discussion by iammaxlichter** [community/community#120305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For some reason, no matter what I do, GitHub Pages is not correctly deploying my website for some reason even though I fit all the criteria for a working website. I just don't know what to do anymore
![Screenshot_19-4-2024_195928_iammaxlichter github io](https://github.com/community/community/assets/107379037/b1ab5ffd-c5a0-471a-add6-cb5234558e98)
![Screenshot_19-4-2024_20224_github com](https://github.com/community/community/assets/107379037/14c1d696-ccc8-450f-afdf-d4b50eaf4342)
![Screenshot_19-4-2024_20210_github com](https://github.com/community/community/assets/107379037/f2f4fd61-2eff-4b37-9fe3-d5c2e2fc0281)

heres my index.html too. I am incredibly frustrated and have been trying to get this to work for the last 5 hours

<!doctype html>
<html lang="en">

<head>

    
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
    <link href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@400;700&display=swap" rel="stylesheet">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" rel="stylesheet">

    <link rel="stylesheet" href="CSSComponents/RequiredComponents/footer.css">
    <link rel="stylesheet" href="CSSComponents/RequiredComponents/navbar.css">
    <link rel="stylesheet" href="CSSComponents/MainComponents/index.css">
    <link rel="stylesheet" href="CSSComponents/ExtraComponents/shareModal.css">
    <link rel="stylesheet" href="CSSComponents/ExtraComponents/growAnimation.css">
    <link rel="stylesheet" href="CSSComponents/ExtraComponents/slideInBottomView.css">
    <link rel="stylesheet" href="CSSComponents/ExtraComponents/slideInLeftView.css">
    <link rel="stylesheet" href="CSSComponents/ExtraComponents/slideInRight.css">

    <link rel="icon" href="Pictures/logo.png" type="image/x-icon">
    <title>Max Lichter</title>

</head>

<body>

    
    <nav id="navbar" class="sticky-top"></nav>
    <div id="mainContent"></div>
    <div id="footer"></div>

    <script>

        // Function to fetch and insert HTML content into specified element
        function fetchAndInsert(url, targetId) {
            fetch(url)
                .then(response => response.text())
                .then(html => {
                    document.getElementById(targetId).innerHTML = html;
                    // If targetId is 'footer', add event listener for copy button click
                    if (targetId === 'footer') {
                        document.getElementById("copyButton").addEventListener("click", copyUrlToClipboard);
                    }

                    handleNavbar();
                });

        }

        // Call fetchAndInsert function for navbar, main content, and footer
        fetchAndInsert('HTMLcomponents/RequiredComponents/navbar.html', 'navbar');
        fetchAndInsert('HTMLcomponents/MainComponents/mainIndex.html', 'mainContent');
        fetchAndInsert('HTMLcomponents/RequiredComponents/footer.html', 'footer');
    </script>

    
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
        integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
        crossorigin="anonymous"></script>

    
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js"
        integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
        crossorigin="anonymous"></script>

    <script src="JSComponents/navbar.js"></script>
    <script src="JSComponents/footer.js"></script>
    <script src="JSComponents/slideInLeftView.js"></script>
    <script src="JSComponents/slideInBottomView.js"></script>
    <script src="JSComponents/slideInRightView.js"></script>

</body>

</html>

**Comment by iammaxlichter** [comment]
Also, the live server in VS code is working fine. I have no console errors either. I just don't know what to do

**Comment by eddysantiagoo** [comment]
@iammaxlichter I'm just checking your project and it must be a file structure problem or permissions, your index components are 3, it is very curious that the response from github pages is 404 in the index of your website, this means that github pages is not finding your required components, I recommend you try netfliy or vercel or remake this site in some other way more easy to maintein and deploy, it is not advisable to use so many fetches to add the components in a project with only html, css and js

If the answer helps you please mark this answer as the correct answer

**Reply by iammaxlichter** [reply]
It’s weird because I made a website called Ally’s Photography that has the same file structure, and it is working fine

https://iammaxlichter.github.io/Allys-Photography

**Reply by eddysantiagoo** [reply]
Yes very interesting. If the project works locally, this could be an issue with the configuration of GitHub Pages or a temporary problem with the GitHub Pages service. Two days ago i tried to deploy an Astro project and also encountered a 404 error
